spec/models/presentation_spec.rb

```diff
@@ -1,5 +1,154 @@
+# spec/models/presentation_spec.rb
 require 'rails_helper'
 
 RSpec.describe Presentation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:person) { FactoryGirl.create(:person) }
+  let(:event)  { FactoryGirl.create(:event) }
+
+  describe 'validate model' do
+    subject { FactoryGirl.create(:presentation) }
+
+    it { is_expected.to validate_presence_of :topic }
+    it { is_expected.to validate_presence_of :duration }
+    it { is_expected.to validate_presence_of :event }
+
+    it { is_expected.to belong_to :person }
+    it { is_expected.to belong_to :event }
+
+    it { is_expected.to be_valid }
+  end
+
+  describe 'presenter' do
+    subject { described_class.new(presentation) }
+
+    context 'when present, with no person' do
+      let(:presentation) do
+        FactoryGirl.attributes_for(:presentation,
+                                   event_id: event.id,
+                                   presenter: 'test')
+      end
+
+      it { is_expected.to be_valid }
+      it { is_expected.to have_attributes(person: nil, presenter: 'test') }
+    end
+
+    context 'when not present, with person' do
+      let(:presentation) do
+        FactoryGirl.attributes_for(:presentation,
+                                   event_id: event.id,
+                                   person_id: person.id)
+      end
+
+      it { is_expected.to have_attributes(person: person, presenter: nil) }
+      it { is_expected.to be_valid }
+    end
+
+    context 'when present, with person' do
+      let(:presentation) do
+        FactoryGirl.attributes_for(:presentation,
+                                   event_id: event.id,
+                                   presenter: 'test',
+                                   person_id: person.id)
+      end
+
+      it { is_expected.to have_attributes(person: person, presenter: 'test') }
+      it { is_expected.to_not be_valid }
+    end
+  end
+
+  describe 'topic' do
```

Comment:
Yeah... I did this because in a previous pull request for event (i think) Gray commented on my tests using context, but only having one side of it in that describe, but having the other side in the validate. In these cases I can remove the context all together.